sdk/core/Azure.Core/samples/Configuration.md

```diff
@@ -44,3 +44,12 @@ SecretClientOptions options = new SecretClientOptions
     Transport = new HttpClientTransport(handler)
 };
 ```
+
+## Configuring a proxy using environment variables
+
+You can also configure a proxy using the following environment variables:
```

Comment:
Although it is implied, should we clarify that these proxy settings would affect all clients in the environment rather than targeting a specific client as demonstrated above?

Reply:
I considered it, but we literally copied the implementation so what .NET supports at the time someone reads it isn't necessarily what we support.